dist/profile/manifests/staticsite.pp

```diff
@@ -91,19 +89,14 @@
   }
 
   file { $beta_docroot:
-    ensure  => link,
+    ensure  => absent,
     replace => false,
     owner   => $deployer_user,
     group   => $deployer_group,
     target  => "${site_root}/archives",
     require => File["${site_root}/archives"],
   }
 
-  exec { 'chown staticsite':
-    command     => "/bin/chown -R ${deployer_user}:${deployer_group} ${site_root}",
-    refreshonly => true,
-  }
-
   apache::vhost { 'beta.jenkins-ci.org':
```

Comment:
I think this vhost is not used anymore and can be remove  with ensure => 'absent'